### ISSUE TITLE
Updating PHP, Snipe referenced old PHP, then I broke stuff

### ISSUE THREAD

**Issue by MSTigers** [snipe/snipe-it#5536]
#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

I broke everything. Not really a bug at this point, but needing help resolving the issue or a little bit of advice.

**Expected behavior**

**Summary:**

During troubleshooting, mistakenly deleted out `composer.lock`. Believe everything else is fixed, need to 1.) figure out how to resolve the deleted `composer.lock` issue, and then 2.) proper method to point Snipe-IT to the new version of PHP.

**Detail:**
In lieu of the [v4.3 update notes](https://github.com/snipe/snipe-it/releases/tag/v4.3.0) I needed to update my PHP from  7.0.30-0ubuntu0.16.04.1 to a newer version.

7.0 is the latest official Ubuntu repository, so I ran `sudo add-apt-repository ppa:ondrej/php` and updated through `sudo apt update && sudo apt upgrade` which worked great. php -v indicated I was now running PHP 7.2.5.

Nothing I did to Snipe would make it show 7.2.5, but always 7.0.30. Tried `sudo update-alternatives --set php` thinking perhaps I was still defaulting over to 7.0.  Figured maybe I just need to uninstall 7.0! `sudo php remove 7.0*` was my downfall. **_Broke. So. Many. Things._** Uninstalled/purged all php, reinstalled php.

After my reinstall of php, I had very similar errors as this guy from [issue 762](https://github.com/snipe/snipe-it/issues/762), so I deleted `composer.lock` before I learned that was a bad idea from [issue 5337](https://github.com/snipe/snipe-it/issues/5337) since I'm not doing Dev (oops!).

Though I was able to find a couple dependencies that were missing when trying to run `php composer.phar install --no-dev --prefer-source`. Manually installed everything that was missing from error after error and eventually got Snipe working again.

Need to make sure I didn't completely, irreparably break it by rm composer.lock, and how to properly get Snipe to recognize my upgraded PHP.

Note: Presently back to 7.0.30 PHP. Also noticed that it may be a better idea to do a 7.1x PHP instead of 7.2 due to #5519 , but that's moot right now I think.

--

**Server (please complete the following information):**
 - Snipe-IT Version   v4.3.0 build 3586 (gec1fa8e90)
 - OS: Ubuntu:  Ubuntu 16.04.4 LTS
 - Web Server: Apache/2.4.18 
 - PHP Version: 7.0.30-0ubuntu0.16.04.1

**Additional context**
- Is this a fresh install or an upgrade?
Upgrade
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
install.sh
- Indicate whether or not you've manually edited any data directly in the database
No manual db edits

**Comment by MSTigers** [comment]
tl;dr

I accidentally (on purpose) deleted composer.lock, am I SOL? If I'm not SOL, how do you make Snipe recognize an updated version of PHP?

**Comment by snipe** [comment]
No, you're not SOL, but you should grab a copy of composer.lock from master so you have one (or do another git pull to replace the version you deleted). Snipe-IT doesn't need to be aware of the version of PHP you're using, just your web server does.

**Comment by MSTigers** [comment]
Git Pull kept telling me I was already up to date. Pulled a copy from master, just wanted to make sure I was able to replace that file like that. Thanks for the tip!

As for the php (went with 7.1), I finally got Snipe to recognize a newer version. `a2enmod php7.1` was needed in addition to  sudo `update-alternatives --set php /usr/bin/php7.1`

> php -v
> PHP 7.1.17-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: May  5 2018 04:55:21) ( NTS )
> Copyright (c) 1997-2018 The PHP Group
> Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
>     with Zend OPcache v7.1.17-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
> 

> Snipe-IT version | v4.3.0 build 3586 (gec1fa8e90)
> -- | --
> Software License | AGPL3
> PHP Version | 7.1.17-1+ubuntu16.04.1+deb.sury.org+1
> Laravel Version | 5.4.35

Looks like I am ready to go. The documentation in the [readme.io](https://snipe-it.readme.io/docs/requirements) points to php.net article that only explains for php 7.0. Given the approaching requirements for 7.1+ this one may be more useful: [link](https://jakelprice.com/article/how-to-upgrade-from-php-70-to-php-71) since it references the ondrej/php repository which will be needed to stay up-to-date for newer Snipe versions for Ubuntu users.
